src/Volume.ts

```diff
@@ -157,146 +127,110 @@ export default class Volume {
   public loadSpec: LoadSpec;
   public imageMetadata: Record<string, unknown>;
   public name: string;
-  public x: number;
-  public y: number;
-  public z: number;
+  // public x: number;
+  // public y: number;
+  // public z: number;
   public channels: Channel[];
   private volumeDataObservers: VolumeDataObserver[];
-  private physicalSize: Vector3;
   public physicalScale: number;
-  public normalizedPhysicalSize: Vector3;
-  public contentSize: Vector3;
-  public contentOffset: Vector3;
+  public physicalSize: Vector3;
+  public normPhysicalSize: Vector3;
+  public normRegionSize: Vector3;
+  public normRegionOffset: Vector3;
   public physicalUnitSymbol: string;
   public tickMarkPhysicalLength: number;
   private loaded: boolean;
   /* eslint-disable @typescript-eslint/naming-convention */
-  public num_channels: number;
-  public channel_names: string[];
+  // public num_channels: number;
+  public channelNames: string[];
   public channel_colors_default: [number, number, number][];
-  public pixel_size: [number, number, number];
+  // public pixel_size: [number, number, number];
   /* eslint-enable @typescript-eslint/naming-convention */
 
   constructor(imageInfo: ImageInfo = getDefaultImageInfo(), loadSpec: LoadSpec = new LoadSpec()) {
     // imageMetadata to be filled in by Volume Loaders
     this.imageMetadata = {};
-    this.contentSize = new Vector3(1, 1, 1);
-    this.contentOffset = new Vector3(0, 0, 0);
+    this.normRegionSize = new Vector3(1, 1, 1);
+    this.normRegionOffset = new Vector3(0, 0, 0);
     this.physicalSize = new Vector3(1, 1, 1);
     this.physicalScale = 1;
-    this.normalizedPhysicalSize = new Vector3(1, 1, 1);
+    this.normPhysicalSize = new Vector3(1, 1, 1);
 
     this.loaded = false;
     this.imageInfo = imageInfo;
     this.name = this.imageInfo.name;
     this.loadSpec = loadSpec;
 
     // clean up some possibly bad data.
-    this.imageInfo.pixel_size_x = imageInfo.pixel_size_x || 1.0;
-    this.imageInfo.pixel_size_y = imageInfo.pixel_size_y || 1.0;
-    this.imageInfo.pixel_size_z = imageInfo.pixel_size_z || 1.0;
+    this.validatePixelSize();
 
-    this.pixel_size = [this.imageInfo.pixel_size_x, this.imageInfo.pixel_size_y, this.imageInfo.pixel_size_z];
-    this.x = this.imageInfo.tile_width;
-    this.y = this.imageInfo.tile_height;
-    this.z = this.imageInfo.tiles;
+    // TODO clean up these unused properties or replace them with whatever seems most sensible
+    // this.pixel_size = [this.imageInfo.pixel_size_x, this.imageInfo.pixel_size_y, this.imageInfo.pixel_size_z];
+    // this.x = this.imageInfo.tile_width;
+    // this.y = this.imageInfo.tile_height;
+    // this.z = this.imageInfo.tiles;
 
-    this.num_channels = this.imageInfo.channels;
+    // this.num_channels = this.imageInfo.channels;
```

Comment:
I've noticed a lot of commented out lines and TODOs. Should this be added as a separate issue ticket?